pkg/csi_driver/node.go

```diff
@@ -93,8 +98,13 @@ func (s *nodeServer) NodePublishVolume(ctx context.Context, req *csi.NodePublish
 
 		// Login credentials
 
-		options = append(options, s.driver.config.Username)
-		options = append(options, s.driver.config.Password)
+		secrets := req.GetNodePublishSecrets()
```

Comment:
We should modify L52 above to make sure we don't log the contents of the secrets.  We may be able to use https://github.com/kubernetes-csi/csi-lib-utils/blob/master/protosanitizer/protosanitizer.go for this

Reply:
Done.